src/test/java/com/alipay/oceanbase/rpc/ObTableClientTest.java

```diff
@@ -2523,4 +2524,36 @@ public void testQueryWithEmptyTable() throws Exception {
             Assert.assertEquals("table name is null", ((IllegalArgumentException) e).getMessage());
         }
     }
+
+    @Test
+    public void testQueryWithScanOrder() throws Exception {
```

Comment:
It seems that this test is a basic test, could we add this test to our CI test?
Add test here ObTableClientITCase, with `obTableClientTest.testQueryWithScanOrder()`

Reply:
This bugfix of observer may not sync to the open source code and we wouldn't add this test case to ObTableClientITCase which may cause ci test failed.